### ISSUE TITLE
birthdays and deaths don't have age options

### ISSUE THREAD

**Issue by nutpantz** [m-i-n-a-r/birday#225]
Birthdays have no stop option for people who have died. It keeps adding age, so you don't know the last age
Death anniversaries have no age option so you don't know how old they were when they died.

So the birthday contact needs a stop aging at date option
And the death anniversaries need a birthday for age at death.

**Comment by nutpantz** [comment]
I am going to disagree. I want to know how old the dead person was when they died, while still celebrating their birth. They do not age after death.

No one ever asks how old Elvis would be today, but some want to know how old he was when he died

**Comment by m-i-n-a-r** [comment]
Yep, it makes sense, but this is also a cultural factor. Where i live (Italy) we often think about the age of the person if he/she was alive today. But i understand your point. I'll see what i can do!
